Routing/AnnotatedRouteControllerLoader.php

```diff
@@ -50,7 +50,7 @@ protected function configureRoute(Route $route, \ReflectionClass $class, \Reflec
         // requirements (@Method)
         foreach ($this->reader->getMethodAnnotations($method) as $configuration) {
             if ($configuration instanceof Method) {
-                $route->setMethods(implode('|', $configuration->getMethods()));
+                $route->setMethods(explode('|', implode('|', $configuration->getMethods())));
```

Comment:
Sorry, but I cannot reproduce your observations. That example gives me the following output:

```
Array
(
    [0] => GET
    [1] => POST
)
```

Which version of the bundle did you use (I tested it with the latest version, 3.0.11)?